packages/edit-site/src/components/global-styles/screen-typeset.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { __experimentalVStack as VStack } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import TypographyVariations from './variations/variations-typography';
+import ScreenHeader from './header';
+import Typeset from './typeset';
+
+function ScreenTypeset() {
+	return (
+		<>
+			<ScreenHeader
+				title={ __( 'Typesets' ) }
+				description={ __(
+					'Font pairings and typographic styling applied across the site.'
+				) }
+			/>
+			<div className="edit-site-global-styles-screen">
+				<VStack spacing={ 7 }>
+					<Typeset />
+					<TypographyVariations title={ __( 'Presets' ) } />
```

Comment:
Let's name this "Typesets" instead of "Presets".  (If we decide to use a different name other than "Typeset" we'll use that here).

Reply:
I'm happy with Typesets or Typeset. Or Font sets and Font set.